tool/tyber/go/go.mod

```diff
@@ -1,8 +1,9 @@
-module berty.tech/berty/tool/tyber/go/v2
```

Comment:
hum, I don't understand why you're doing this here?

Reply:
it was breaking the linux build with `/v2`

Reply:
are you using astilectron latest version? because it's similar with gomobile when gomobile was 1 or 2 versions below go master and not supporting go modules well at that time

Reply:
I tried updating astilectron and changing back to `/v2`, now I have a different error:
```
2021/05/21 08:43:05 Executing go build -ldflags -X "main.AppName=Tyber" -X "main.BuiltAt=2021-05-21 08:43:05.458300063 +0200 CEST m=+11.682350494" -X "main.VersionAstilectron=0.48.0" -X "main.VersionElectron=11.4.3" -o /home/norman/go/src/berty.tech/berty/tool/tyber/output/linux-amd64/binary berty.tech/berty/tool/tyber/go
2021/05/21 08:43:05 bundling failed: bundling for environment linux/amd64 failed: building failed: no required module provides package berty.tech/berty/tool/tyber/go; to add it:
	go get berty.tech/berty/tool/tyber/go
```
I couldn't find where I can override what package astilectron targets, not sure why it's not `/v2`

Reply:
The problem with astilectron is also that it will use real path on the file system to build the electron app.
So if your gomodule path is `berty.tech/berty/v2/tool/tyber` and the path on your filesystem is `~/go/src/berty.tech/berty/tool/tyber` (without v2), it won't work.

Reply:
let's remove the `/v2` for now, I'll check later if I can unify the gomodule config